bundles/org.openhab.binding.spotify/README.md

```diff
@@ -214,6 +214,7 @@ sitemap spotify label="Spotify Sitemap" {
     Default   item=spotifyTrackPlayer   label="Player"
     Switch    item=spotifyDeviceShuffle label="Shuffle mode:"
     Text      item=spotifyTrackRepeat   label="Repeat mode: [%s]"
+    Selection item=spotifyTrackRepeat   label="Repeat mode: [%s]" mappings=[off="OFF",context="all repeat",track="track repeat"]
```

Comment:
This should not be necessary since the channel already contains an options list. The item above here could actually be a `Selection` instead of `Text`.

Reply:
I agree with @Hilbrand. So, closing this.